src/components/screens/SendScreen/screens/TransactionProcessingScreen.tsx

```diff
@@ -0,0 +1,330 @@
+          <View className="flex-row items-center gap-[4px]">
+            <Icon.ArrowCircleUp size={16} color={themeColors.text.secondary} />
+            <Text sm medium secondary>
+              {dateTimeDisplay}
+            </Text>
+          </View>
+        </View>
+      </View>
+
+      {/* Gray box with transaction data */}
+      <View className="bg-background-secondary rounded-[16px] p-[24px] gap-[12px]">
+        {/* First row: Token amount and Asset icon */}
+        <View className="flex-row items-center justify-between">
+          <View>
+            <Text xl medium primary>
+              {formatAssetAmount(tokenValue, selectedBalance?.tokenCode)}
+            </Text>
+            <Text md medium secondary>
+              {formatFiatAmount(
+                new BigNumber(tokenValue).times(
+                  selectedBalance?.currentPrice || 0,
+                ),
+              )}
+            </Text>
+          </View>
+          {selectedBalance && <AssetIcon token={selectedBalance} size="lg" />}
+        </View>
+
+        {/* Second row: Centered circle with chevron */}
+        <View>
+          <View className="w-[32px] h-[32px] rounded-full bg-tertiary justify-center items-center border border-gray-6">
+            <Icon.ChevronDownDouble
+              size={20}
+              color={themeColors.foreground.primary}
+            />
+          </View>
+        </View>
+
+        {/* Third row: Recipient address info and avatar */}
+        <View className="flex-row items-center justify-between">
+          <View>
+            <Text xl medium primary>
+              {slicedAddress}
+            </Text>
+            <Text md medium secondary>
+              First time send
+            </Text>
+          </View>
+          <Avatar size="lg" publicAddress={address} />
+        </View>
+      </View>
+
+      {/* Transaction details box with border */}
+      <View className="rounded-[16px] p-[24px] gap-[12px] bg-background-primary border-gray-6 border">
+        {/* Status row */}
+        <View className="flex-row items-center justify-between">
+          <View className="flex-row items-center gap-[8px]">
+            <Icon.ClockCheck size={16} color={themeColors.foreground.primary} />
+            <Text md medium secondary>
+              Status
+            </Text>
+          </View>
+          <Text md medium style={{ color: themeColors.status.success }}>
+            Success
+          </Text>
+        </View>
+
+        {/* Rate row */}
+        <View className="flex-row items-center justify-between">
+          <View className="flex-row items-center gap-[8px]">
+            <Icon.Divide03 size={16} color={themeColors.foreground.primary} />
+            <Text md medium secondary>
+              Rate
+            </Text>
+          </View>
+          <Text md medium>
+            {`1 ${selectedBalance?.tokenCode} ≈ ${selectedBalance?.currentPrice ? (1 / Number(selectedBalance.currentPrice)).toFixed(3) : "0.000"} USDC`}
```

Comment:
I don't think this "Rate" row makes sense for the Send flow. We should either remove it or replace it with something like "Amount sent    {tokenAmount} {tokenCode}".

This "Rate" row makes more sense for the Swap flow where we are swapping one asset for the other so we can display the swap rate for the user.

I've posted the same question here on Figma: https://www.figma.com/design/KwkHXQxbNmDllwermJtnRu?node-id=3707-85996#1239107471